py3status/modules/deadbeef.py

```diff
@@ -67,56 +79,48 @@ def _is_running(self):
 
     def deadbeef(self):
         color = self.color_stopped
-        artist = isplaying = length = playback_time = title = \
-            tracknumber = year = ""
+        status = ''
```

Comment:
This is wrong.  status should be `{}` or possibly `None`.  This may be the cause of your issue.